src/main/cpp/Robot.cpp

```diff
@@ -17,13 +17,18 @@ void Robot::RobotPeriodic() {
   {
         frc::DataLogManager::Log( e.what());
   }
+  m_container.LogDashboard();
 }
 
 void Robot::DisabledInit() {}
 
 void Robot::DisabledPeriodic() {}
 
-void Robot::DisabledExit() {}
+void Robot::DisabledExit() {
+  if (!CONSTANTS::IN_MATCH) {
```

Comment:
Instead of doing this, use `frc::DriverStation::IsFMSAttached()` instead. The FMS is a system that the robot connects to in comp so the refs can disable it and auto timings work properly.